src/extensions/surveys/surveys-utils.tsx

```diff
@@ -645,6 +652,10 @@ export const hasEvents = (survey: Survey): boolean => {
 }
 
 export const canActivateRepeatedly = (survey: Survey): boolean => {
+    if (survey.schedule === SurveySchedule.Always && survey.type === SurveyType.Widget) {
```

Comment:
Can we add tests for this? ideally all the new code that is testable, we'd add tests to cover more of the surveys codebase

Reply:
@marandaneto i agree, just added to this function and also to the one it depends too